
We do not need this conditional rename on sparc.

  - Rene Rebe <rene@exactcode.de>

--- ./xc/programs/Xserver/hw/xfree86/os-support/linux/lnx_io.c.orig	2004-11-08 20:17:04.108288696 +0100
+++ ./xc/programs/Xserver/hw/xfree86/os-support/linux/lnx_io.c	2004-11-08 20:19:01.000000000 +0100
@@ -74,7 +74,7 @@
 #endif
 
 /* Deal with spurious kernel header change */
-#if defined(LINUX_VERSION_CODE) && defined(KERNEL_VERSION)
+#if !defined(__sparc__) && defined(LINUX_VERSION_CODE) && defined(KERNEL_VERSION)
 # if LINUX_VERSION_CODE >= KERNEL_VERSION(2,5,42)
 #  define rate period
 # endif
--- ./xc/programs/Xserver/hw/xfree86/os-support/linux/lnx_kbd.c.orig	2004-11-08 20:19:39.792621080 +0100
+++ ./xc/programs/Xserver/hw/xfree86/os-support/linux/lnx_kbd.c	2004-11-08 20:20:06.000000000 +0100
@@ -101,7 +101,7 @@
 #endif
 
 /* Deal with spurious kernel header change */
-#if defined(LINUX_VERSION_CODE) && defined(KERNEL_VERSION)
+#if !defined(__sparc__) && defined(LINUX_VERSION_CODE) && defined(KERNEL_VERSION)
 # if LINUX_VERSION_CODE >= KERNEL_VERSION(2,5,42)
 #  define rate period
 # endif
